pandas/tests/io/test_sql.py

```diff
@@ -1781,7 +1781,7 @@ def check(col):
                 )
 
         # GH11216
-        df = read_sql_query("select * from types", self.conn)
+        df = read_sql_table("types", self.conn)
```

Comment:
Hmm this test has been passing on our CI with postgres, and from https://github.com/pandas-dev/pandas/pull/11216 it was an intentional choice to test `read_sql_query` instead of `read_sql_table`

Reply:
Which platforms is it being tested on?  I'm working on Windows, and it could be that something about my setup is preventing the timezone information from getting into the pandas DataFrame.  I could try switching to Ubuntu.

Reply:
Ah that could be it. This SQL test runs on Ubuntu + Postgres

Reply:
It turns out that the test failure was due to the postgres database timezone setting, not the platform.  To get it to pass on either Windows or Ubuntu, I needed to change the database timezone to UTC.  If there is an issue to be addressed when the timezone is not UTC, I'm not sure how best to handle it, so I will stop working on this PR.